common/lib/driver-definitions/src/storage.ts

```diff
@@ -49,7 +49,8 @@ export interface IDeltaStorageService {
         tenantId: string,
         id: string,
         from: number, // inclusive
-        to: number // exclusive
+        to: number, // exclusive
+        fetchReason?: string,
```

Comment:
it would be great to add @param comment describing expectations and where it ends up (i.e. no PII)

Reply:
done